include/coap/net.h

```diff
@@ -279,6 +280,18 @@ int coap_context_set_psk( coap_context_t *context, const char *hint,
 int coap_context_set_pki(coap_context_t *context,
                            coap_dtls_pki_t* setup_data);
 
+/**
+ * Set the context keepalive timer for sessions.
+ * A keepalive message will be sent after if a session has been inactive,
+ * i.e. no packet sent or received, for the given number of seconds.
+ * For reliable protocols, a PING message will be sent. If a PONG has not
+ * been received before the next PING is due to be sent, the session will
+ * considered as disconnected.
+ * For unreliable protocols, an empty packet is sent
```

Comment:
Actually, the current changes do not do this for unreliable protocols.

Reply:
Your are correct. I intended to do so initially, but then I realized it requires more changes since right now receiving a reset causes the client session to be shut down. I was also concerned that some implementations may as well clear the client state on their side when sending a reset.
I would propose to remove the line about unreliable protocols and push a separate pull request for the confirmable empty message / elicited reset sequence.

Reply:
Postponing doing the unreliable protocol and doing that as a separate PR works for me as it keeps the code changes separate.

Reply:
OK, I fixed the documentation for the function in the header.